Mathlib/Combinatorics/SimpleGraph/AdjMatrix.lean

```diff
@@ -279,6 +279,20 @@ theorem adjMatrix_pow_apply_eq_card_walk [DecidableEq V] [Semiring α] (n : ℕ)
       simp at hxy
 #align simple_graph.adj_matrix_pow_apply_eq_card_walk SimpleGraph.adjMatrix_pow_apply_eq_card_walk
 
+/-- The sum of the identity, the adjacency of `G`, and the adjacency of `Gᶜ` yeilds all ones. -/
+theorem one_add_adjMatrix_add_compl_adjMatrix_eq_allOnes [DecidableEq V] :
+    1 + G.adjMatrix ℕ + (G.adjMatrix ℕ).compl = fun _ _ ↦ 1 := by
```

Comment:
```suggestion
    1 + G.adjMatrix ℕ + (G.adjMatrix ℕ).compl = Matrix.of fun _ _ ↦ 1 := by
```

Reply:
Suggestion committed in addition to rewriting `of_apply` to make it work.  Thanks!